system-parachains/bridge-hubs/bridge-hub-polkadot/src/bridge_to_ethereum_config.rs

```diff
@@ -116,6 +116,10 @@ parameter_types! {
 			version: [4, 0, 0, 0], // 0x04000000
 			epoch: 269568,
 		},
+		electra: Fork {
+			version: [5, 0, 0, 0], // 0x05000000
+			epoch: todo!("TODO: @claravanstaden please check"),
+		},
```

Comment:
@claravanstaden can you please check this?

Reply:
Electra on mainnet will only be after 25 April. Setting this to an epoch in 2083 for now.

Reply:
It sets the Electra epch in the year [2083 (epoch `5000000`)](https://sepolia.beaconcha.in/epoch/5000000)